packages/phrases/src/locales/en.ts

```diff
@@ -118,7 +118,7 @@ const translation = {
     },
     application_details: {
       back_to_applications: 'Back to Applications',
-      check_help_guide: 'Check Help Guide',
+      check_help_guide: 'Check Guide',
```

Comment:
@demonzoo @xiaoyijun this key should update accordingly

Reply:
np. https://github.com/logto-io/logto/pull/1390